src/glyphs.cpp

```diff
@@ -548,10 +548,6 @@ void RenderSDF(glyph_info &glyph,
         return;
     }
 
-    FT_Glyph ft_glyph = nullptr;
-    ft_glyph_guard glyph_guard(&ft_glyph);
-    if (FT_Get_Glyph(ft_face->glyph, &ft_glyph)) return;
```

Comment:
Okay, I think the original intent here may have been to check that `FT_Get_Glyph` doesn't error before trying `FT_Outline_Decompose`, but this may not be necessary given that we we're just silently skipping the glyph if either fail.

Makes more sense now why we wouldn't need to free the glyph with `FT_Done_Glyph` if we're not getting it with `FT_Get_Glyph` in the first place.